functions.php

```diff
@@ -278,7 +281,7 @@ public function engaging_network_id_show_settings( $args ) {
 	 */
 	public function cookies_show_settings( $args ) {
 		$cookies_text = get_option( 'cookies_field', '' );
-		$args = [
+		$args         = [
```

Comment:
There's quite a bit of un-related spacing updates in this PR, I'd prefer to see just the changes that are required and any directly-related code.

Reply:
Spacing reverted.

Reply:
Thinking about it, I'm not sure how the spacing got like that. I didn't go through the file correcting spacing or anything, and I don't recall running phpcbf. Maybe phpstorm did it automatically somehow? Then it might be evidence of the revision tree getting out of order. I'll watch it closely on my next commit.

Reply:
Looks like my attempt to fix it just made the next commit messier. I'll leave it and take note of that for the future.